javagen/src/main/java/com/azure/autorest/model/javamodel/JavaClass.java

```diff
@@ -199,6 +204,64 @@ public final void javadocComment(int wordWrapWidth, Consumer<JavaJavadocComment>
         contents.javadocComment(wordWrapWidth, commentAction);
```

Comment:
Should we remove other old `javadocComment` overload, to avoid anyone calling the unwanted method?